api/api.go

```diff
@@ -23,11 +23,3 @@ type Client interface {
 	Export(context.Context) ([]byte, error)
 	PublicSSHKey(ctx context.Context, regenerate bool) (ssh.PublicKey, error)
 }
-
-// API for daemons connecting to an upstream service
-type Upstream interface {
```

Comment:
The methods belong somewhere in the service because iirc there's neither a reference to nor an implementation of this interface anywhere in this repo.